environment.yml

```
@@ -5,6 +5,6 @@ channels:
 dependencies:
   - python=3.7
   - jax=0.3.14
-  - jaxlib=0.3.14
+  - jaxlib=0.3.14=*cpu*
```

nit: maybe
```suggestion
  - jaxlib=0.3.14=cpu*
```
(since `cpu`/`cuda` is at the beginning of the build description string)